projects/mdl-js.md

```diff
@@ -0,0 +1,72 @@
+# Project Name
+
+mdl-js
+
+# Preferred Maturity Level
+
+Labs
+
+# Project Description
+
+This is a Typescript(Javascript) implementation of the [ISO 18013-5:2021](https://www.iso.org/standard/69084.html) (MDL) specification.
+
+There is only one mdl implementation from Auth0 labs, but they don't support React Native which is most crucial for mobile wallets. This project aims to provide a platform agnostic implementation of the MDL specification.
+
+I guess the main reason for not providing a React Native is that core dependencies like CBOR, COSE and X509 libraries are not available for React Native. So I decided to make them all platform agnostic.
+
+To provide mdl features to Biggest Wallet SDKs like credo or veramo, direction of the project will be:
+
+- Provide a platform agnostic implementation of the MDL specification
+- Bring your own crypto
+- Modular approach
+- Provide platform sepcific helper implementations for developers to use easily
+
+# Alignment with the OpenWallet Foundation Mission
+
+ISO 18013-5 (MDL) is one of the most important standards for digital wallets. The architecture reference framework of the EU also includes MDL.
+
+# Code of Conduct
+
+[OpenWallet Foundation code of conduct](https://tac.openwallet.foundation/governance/code-of-conduct/)
+
+# TAC Sponsor
+
+@aceshim Ace Shim
+
+# Project License
+
+Apache 2.0
+
+# Source Control
+
+https://github.com/lukasjhan/mdoc
```

Comment:
I noticed that DCO does not exist on the current repository. See [Fixing DCO](https://github.com/openwallet-foundation/project-proposals?tab=readme-ov-file#fixing-dco-on-existing-repositories) for more information. We will need this corrected prior to bringing this code into the OpenWallet Foundation. We can also create a new repo and have you squash/merge if that is preferred.

Reply:
I'll fix DCO :)

Reply:
@tkuhrt I just finished add sign-off to all my commit :)